src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 4;
+    private static final int MAX_VAL = 100;
+    private ColorSupplier supplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        switch (figureNumber) {
+            case 1:

```

Comment:
```suggestion
            case 0:
```
нумерація case блоків повинна починатись з 0 і йти до 3, адже `figureNumber` може дорівнювати від 0 до 3